### PR TITLE
Use new Meta/deprecate_function in sysdata

### PR DESCRIPTION
**Pull request by guiniol**
Try to use the new Meta/deprecated function for deprecated config args. Tox doesn't like it.

### CODE REVIEW

File: py3status/modules/sysdata.py [hunk 1]
```diff
@@ -140,27 +134,42 @@ class Py3status:
...
+        def deprecate_function(config):
+            # support old thresholds
+            return {
+                    'thresholds': [
+                        (0, 'good'),
+                        (config.get('med_threshold', 40), 'degraded'),
+                        (config.get('high_threshold', 75), 'bad'),
+                        ]
+                    }
+            deprecated = {
```
Comment (tobes) [inline py3status/modules/sysdata.py]: This is on the wrong indention level. Should be part of Meta not the function

Comment (guiniol) [inline py3status/modules/sysdata.py]: Of course... Should be good now.

**Comment by ultrabug** [comment]
plz rebase mate

**Comment by guiniol** [comment]
@ultrabug done

**Comment by tobes** [comment]
I think one good change (should have been in #541 really) would be to alter the default format to 
```
    format = "[\?color=cpu CPU: {cpu_usage}%], " \
        "[\?color=mem Mem: {mem_used}/{mem_total} GB ({mem_used_percent}%)]"
```

That way users automatically get nicer behavior and I think the change is an improvement to the current output rather than a regression.  But it doesn't stop this being merged

**Comment by guiniol** [comment]
> That way users automatically get nicer behavior and I think the change is an improvement to the current output rather than a regression. But it doesn't stop this being merged

I even added `max_cpu_mem` to get the original behaviour ^^

**Comment by tobes** [comment]
> I even added max_cpu_mem to get the original behaviour ^^

I approve of this approach and generally I do not think output should change.  But I also think users should get the advantage of the new 'works like it should' version without having to do anything in this instance.  The fact they can still have the 'broken' output if they insist is a nice touch.

**Comment by guiniol** [comment]
I can make a "better defaults" PR. Or add it to this one if you want.

**Comment by tobes** [comment]
> I can make a "better defaults" PR. Or add it to this one if you want.

I'm happy either way but maybe do as a separate one just in case @ultrabug doesn't agree

**Comment by guiniol** [comment]
No problem. I'll open another one once this one is in.

**Comment by guiniol** [comment]
@tobes opened #558

**Pull request merged**